docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -252,11 +252,4 @@ Similarly, to load {kib} settings from a file, you overwrite `/usr/share/kibana/
 See the product-specific documentation for information about running a specific Elastic product in Docker:
 
 * {ref}/docker.html[Install {es} with Docker]
-* {apm-server-ref}/running-on-docker.html[Running APM Server on Docker]
-* {auditbeat-ref}/running-on-docker.html[Running {auditbeat} on Docker]
-* {filebeat-ref}/running-on-docker.html[Running {filebeat} on Docker]
-* {heartbeat-ref}/running-on-docker.html[Running {heartbeat} on Docker]
-* {kibana-ref}/docker.html[Running {kib} on Docker] 
-* {logstash-ref}/docker.html[Running {ls} on Docker]
-* {metricbeat-ref}/running-on-docker.html[Running {metricbeat} on Docker]
-* {packetbeat-ref}/running-on-docker.html[Running {packetbeat} on Docker]
```

Comment:
The [Getting started](https://www.elastic.co/guide/en/elastic-stack-get-started/current/get-started-elastic-stack.html) topic seems to focus on Elasticsearch, Kibana, and Beats. So maybe we just list the links for Elasticsearch, Kibana, and the Elastic Agent? It looks like the `elastic-agent-container.html` file hasn't been moved yet to the Observability guide.

Reply:
> It looks like the elastic-agent-container.html file hasn't been moved yet to the Observability guide.

I'm not planning to move  to elastic-agent-container.html to the obs guide because the docs are needed for other use cases (not just observability). 

I'm on the fence about removing these links because I think it's still hard for users to find all the content they need about running on containers (though I agree that Beats aren't really for "getting started").  We could group them under  a section called "Legacy shippers"? 

I'd be curious to hear what @lcawl thinks about the related links.